frontend/src/app/shared/components/dynamic-forms/typings.d.ts

```diff
@@ -15,16 +16,34 @@ export interface IOPFormlyFieldSettings extends FormlyFieldConfig {
   [key:string]:any;
 }
 
-export interface IOPFormlyTemplateOptions extends FormlyTemplateOptions {
+export interface IOPFormlyTemplateOptions extends FormlyFieldProps {
+  type?:'checkbox' | 'number' | 'password' | 'selectInput' | 'text';
+  name?:string;
   property?:string;
-  label?:string;
   hasDefault?:boolean;
   fieldGroup?:string;
   isFieldGroup?:boolean;
   collapsibleFieldGroups?:boolean;
   collapsibleFieldGroupsCollapsed?:boolean;
   helpTextAttributeScope?:string;
   showValidationErrorOn?:'change' | 'blur' | 'submit' | 'never';
+  payloadValue?:{ href?:string|null };
+  rtl?:boolean;
+  locale?:string;
+  bindLabel?:string;
+  bindValue?:string;
+  searchable?:boolean;
+  editorType?:ICKEditorType;
+  noWrapLabel?:boolean;
+  virtualScroll?:boolean;
+  clearOnBackspace?:boolean;
+  clearSearchOnAdd?:boolean;
+  hideSelected?:boolean;
+  text?:Record<string, string>;
+  typeahead?:boolean;
+  inlineLabel?:boolean;
+  clearable?:boolean;
+  multiple?:boolean;
```

Comment:
`FormlyFieldConfig` is now a generic interface with a `Props` type parameter. As stated in the associated commit, this could (and perhaps should) be reworked as a an interface hierarchy (+ discriminated unions), defining only the applicable properties for a given `OPInputType` or template type. Added [op#60912] (https://community.openproject.org/wp/60912) to track this.

Reply:
We don't want to spend any time into Formly, as the project form will be replaced by Primer at some point in time